scripts/zephyr_module.py

```diff
@@ -299,9 +299,9 @@ def git_revision(path):
     meta['workspace'] = {}
     workspace_dirty |= zephyr_dirty
 
-    if west_projects is not None:
+    if west_projs is not None:
         from west.manifest import MANIFEST_REV_BRANCH
-        projects = west_projects['projects']
+        projects = west_projs['projects']
```

Comment:
I believe line 308 must now be changed as this is no longer projects but array of paths:
https://github.com/zephyrproject-rtos/zephyr/blob/d5f6d90697dc95e0eff48b1ba8e616027629aa04/scripts/zephyr_module.py#L308

Reply:
same story here: https://github.com/zephyrproject-rtos/zephyr/blob/d5f6d90697dc95e0eff48b1ba8e616027629aa04/scripts/zephyr_module.py#L315-L316

Reply:
Fixed

Reply:
you forgot line 316:
https://github.com/zephyrproject-rtos/zephyr/blob/8f19466476a3f185f12b66f559e4200519a1bdc9/scripts/zephyr_module.py#L316

Reply:
I rewrote this